roles/common/tasks/python.yml

```diff
@@ -4,8 +4,8 @@
        version={{ item.rev }} accept_hostkey=True
   with_items: common.python_extra_packages
 
-- name: freeze eventlet to 0.15.2 for nova to work
-  pip: name=eventlet version=0.15.2
+- name: pip install eventelt
```

Comment:
This spelling is wrong. Fix it up and we'll merge without waiting for another test.